mautic/mautic
# Fixed segment lookup_id types (i.e. owner)

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #2892 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Add a new segment and add owner
2. Add anything to the value and save
3. Note the validation failure

#### Steps to test this PR:
1. Repeat and this time the lookup should function. Click on a name and save. The segment will save.
